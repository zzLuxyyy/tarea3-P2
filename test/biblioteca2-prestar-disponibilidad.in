crearBiblioteca
crearSocio 12345678 Jim Dear 10 03 2004
agregarSocioBiblioteca
crearSocio 23456789 Lucia Perez 10 03 2010
agregarSocioBiblioteca
crearFecha 01/01/1966
crearLibro 411222335 "Todos los fuegos el fuego" "Julio" "Cortázar" "Ocho cuentos" 1
agregarLibroBiblioteca
crearFecha 01/01/1961
crearLibro 421233446 "La ratesa" "Günter" "Grass" "Una alegoría" 2
agregarLibroBiblioteca
disponibleLibroBiblioteca 411222335
disponibleLibroBiblioteca 421233446
crearFecha 01/03/1970
prestarLibroBiblioteca 12345678 411222335
imprimirPrestamosBiblioteca
disponibleLibroBiblioteca 411222335
disponibleLibroBiblioteca 421233446
liberarBiblioteca
Fin