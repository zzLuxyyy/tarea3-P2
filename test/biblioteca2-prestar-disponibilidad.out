1>Biblioteca creada exitosamente.
2>Socio de ci 12345678 creado exitosamente.
3>Socio de CI 12345678 agregado a la biblioteca.
4>Socio de ci 23456789 creado exitosamente.
5>Socio de CI 23456789 agregado a la biblioteca.
6>7>8>Libro de ISBN 411222335 agregado a la biblioteca.
9>10>11>Libro de ISBN 421233446 agregado a la biblioteca.
12>El libro de ISBN 411222335 está disponible.
13>El libro de ISBN 421233446 está disponible.
14>15>Préstamo del libro 411222335 al socio 12345678 registrado.
16>LDE Préstamos:
Préstamo de libro Todos los fuegos el fuego a Jim Dear.
1/3/1970
No retornado
17>El libro de ISBN 411222335 NO está disponible.
18>El libro de ISBN 421233446 está disponible.
19>Biblioteca liberada exitosamente.
20>Fin.
